### PR TITLE
Added grain for osdisk stats

### PR DESCRIPTION
**Pull request by xnaveira**

### CODE REVIEW

File: grains/osdisk.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import os  
+  
+def get_osdisk_stats():
+"""
+	Calculates and resturns the disk used, available and total capacity
```
Comment (nmadhok) [inline grains/osdisk.py]: Should be returns instead of resturns

File: grains/osdisk.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import os  
+  
+def get_osdisk_stats():
+"""
```
Comment (nmadhok) [inline grains/osdisk.py]: Change this to three single quotes instead '''

File: grains/osdisk.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	Calculates and resturns the disk used, available and total capacity
+	in gigabytes. Calculations code from:
+	http://www.stealthcopter.com/blog/2009/09/python-diskspace/
+"""
```
Comment (nmadhok) [inline grains/osdisk.py]: Change this to three single quotes too '''

Comment (xnaveira) [inline grains/osdisk.py]: Done. I fixed indentation and trailing spaces as well, I wrote that in the server's vim and became a mess :)

**Pull request merged**